packages/server/src/queries/data-services/portfolio-over-time.ts

```diff
@@ -0,0 +1,27 @@
+import { apiClient } from "@osmosis-labs/utils";
+
+import { NUMIA_BASE_URL } from "../../env";
+
+export interface PortfolioOverTimeResponse {
+  timestamp: string;
+  usd: number;
+}
+
+export async function queryPortfolioOverTime({
+  address,
+  range,
+}: {
+  address: string;
+  range: string;
+}): Promise<PortfolioOverTimeResponse[]> {
+  const url = new URL("/users/portfolio/over_time", NUMIA_BASE_URL);
+
+  url.searchParams.append("address", address);
+  url.searchParams.append("range", range);
+
+  const headers = {
+    Authorization: `Bearer ${process.env.NEXT_PUBLIC_NUMIA_API_KEY}`,
```

Comment:
Like we did with `NUMIA_BASE_URL`, could we import `NEXT_PUBLIC_NUMIA_API_KEY` from that file as well?

Also, for an API key, we'd likely want to remove `NEXT_PUBLIC_` as that would leak the env var into the client (that's the signifier next.js uses to make env vars available to client)